### PR TITLE
Add option to start Chrome

### PR DESCRIPTION
**Pull request by akirk**
Adds a new option so that you can decide which browser to use.

### CODE REVIEW

**Review by psrpinto — APPROVED** [review]
Makes sense!

File: package.json [hunk 1]
```diff
@@ -2,9 +2,11 @@
 	"scripts": {
 		"build": "webpack",
 		"watch": "webpack --watch --env target=firefox",
-		"start": "concurrently \"npm:watch\" \"npm:web-ext:run:firefox\"",
+		"start": "npm run start:firefox",
+		"start:firefox": "concurrently \"npm:watch\" \"npm:web-ext:run:firefox\"",
+		"start:chrome": "concurrently \"npm:watch\" \"npm:web-ext:run:chrome\"",
 		"start:noreload": "npm run web-ext:run:firefox -- --no-reload",
-		"web-ext:run:firefox": "web-ext -s build/firefox run --target firefox-desktop",
+		"web-ext:run:firefox": "web-ext -s build/firefox run --target firefox-desktop --pref toolkit.legacyUserProfileCustomizations.stylesheets=true",
```
Comment (psrpinto) [inline package.json]: I'm curious, what does this allow you to do? I assume it's related to the `userChrome.css` you mentioned to me earlier.

**Review by psrpinto — CHANGES_REQUESTED** [review]
Actually, this won't work currently because the `watch` only builds firefox

File: package.json [hunk 2]
```diff
@@ -2,9 +2,11 @@
 	"scripts": {
 		"build": "webpack",
 		"watch": "webpack --watch --env target=firefox",
-		"start": "concurrently \"npm:watch\" \"npm:web-ext:run:firefox\"",
+		"start": "npm run start:firefox",
+		"start:firefox": "concurrently \"npm:watch\" \"npm:web-ext:run:firefox\"",
```
Comment (psrpinto) [inline package.json]: I guess we need to extract the `--env target=firefox` into here, and change it to `chrome` for the `start:chrome` script.

(Suggestion not tested)

```suggestion
		"start:firefox": "concurrently \"npm:watch -- --env target=firefox\" \"npm:web-ext:run:firefox\"",
```

File: package.json (hunk 1 above)
Comment (akirk) [reply package.json]: Oh, yes, sorry. It's an accidental change. I'll remove it.

**Review by psrpinto — APPROVED** [review]
Should be good to go now

**Pull request merged**